crates/loader/src/bindle/config.rs

```diff
@@ -49,6 +49,8 @@ pub struct RawWasmConfig {
     pub allowed_http_hosts: Option<Vec<String>>,
     /// Optional list of key-value stores the component is allowed to use.
     pub key_value_stores: Option<Vec<String>>,
+    /// Optional list of sqlite databases the component is allowed to use.
```

Comment:
I don't know if this counts as human readable text, but if it does, then SUPER DUPER NIT we should follow the project by using "SQLite" rather than "sqlite."